server/services/admin/signups/hooks/deleteSignup.js

```diff
@@ -3,14 +3,22 @@ module.exports = () => (hook) => {
     const id = hook.id;
     console.log(id)
     return models.signup
-        .destroy({
+        .findOne({
             where: {
                 id
             },
         })
         .then((res) => {
             hook.result = res;
-            return hook;
+            return models.signup
```

Comment:
```suggestion
            return models.signup({
              where: { id }
            })})
            .then(() => hook)
```

Reply:
This could avoid unnecessary nested chains

Reply:
actually, if we always want to return hook maybe use .finally instead of catch & avoid .then(() => hook) also

Reply:
Fixed (didn't test though :D )